src/Bridge/Doctrine/Orm/Metadata/Property/DoctrineOrmPropertyMetadataFactory.php

```diff
@@ -57,14 +56,7 @@ public function create(string $resourceClass, string $property, array $options =
         foreach ($identifiers as $identifier) {
             if ($identifier === $property) {
                 $propertyMetadata = $propertyMetadata->withIdentifier(true);
-                if ($doctrineClassMetadata instanceof ClassMetadataInfo) {
```

Comment:
Why this change ? It looks wrong to me (`isIdentifierNatural` isn't part of the interface).